addons/html_builder/static/src/core/drop_zone_plugin.js

```diff
@@ -437,13 +439,14 @@ export class DropZonePlugin extends Plugin {
         { toInsertInline, isContentInIframe = true } = {}
     ) {
         const isIgnored = (el) => el.matches(".o_we_no_overlay") || !isVisible(el);
-        const hookEls = [];
+        let hookEls = [];
         for (const parentEl of selectorChildren) {
             const validChildrenEls = [...parentEl.children].filter((el) => !isIgnored(el));
             hookEls.push(...validChildrenEls);
             parentEl.prepend(this.createDropzone(parentEl));
         }
         hookEls.push(...selectorSiblings);
+        hookEls = hookEls.filter((el) => !el.hasAttribute?.("data-selection-placeholder"));
```

Comment:
Why not filter in the already existing filter above? Also not sure this plugin should be aware of the existence of the placeholder one, and not even sure it should be fitered out in the first place. Why not drop between two contenteditable=false? I would personally see it as a feature.